<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="mono">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Monoalfabetyczne</string>
   </property>
  </widget>
  <widget class="QPushButton" name="macierz">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Transpozycja</string>
   </property>
  </widget>
  <widget class="QPushButton" name="poli">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Polilfabetyczne</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Szyfrowanie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="base64">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Base64</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hamming">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Hamming</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rsa">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>RSA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rsa_plik">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>RSA - Plik</string>
   </property>
  </widget>
  <widget class="QPushButton" name="des">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>DES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rc4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>RC4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
